rclcpp/src/rclcpp/subscription_base.cpp

```diff
@@ -515,3 +534,14 @@ SubscriptionBase::get_content_filter() const
 
   return ret_options;
 }
+
+
+// DYNAMIC TYPE ==================================================================================
+bool
+SubscriptionBase::take_dynamic_message(
+  rclcpp::dynamic_typesupport::DynamicMessage & /*message_out*/,
+  rclcpp::MessageInfo & /*message_info_out*/)
+{
+  RCLCPP_WARN(rclcpp::get_logger("rclcpp"), "take_dynamic_message stubbed out for now.");
```

Comment:
Let's just throw an exception here, because there is no way this should happen.

Reply:
https://github.com/ros2/rclcpp/pull/2165/commits/331b7904ce75c222012e53c038e8a6d0ab3e7242